src/main/java/nl/altindag/log/LogCaptor.java

```diff
@@ -223,4 +224,14 @@ public void clearLogs() {
         listAppender.list.clear();
     }
 
+    public void tearDown() {
+        listAppender.stop();
+        logger.detachAppender(listAppender);
+    }
+
+    @Override
+    public void close() {
+        tearDown();
```

Comment:
Why do you call here the teardown method? Isn't it better to move the body of the teardown method to the close method?

Reply:
Sounds fair)
Done